ccf_transaction_processor/scripts/start_ccf_network.sh

```diff
@@ -48,6 +48,12 @@ sleep 5
 echo generate the ledger authority
 try ${F_SERVICEHOME}/bin/generate_ledger_authority.py --logfile __screen__ --loglevel WARNING
 
+#Set the enclave attestation policy while operating under SGX SIM mode. When operating in the HW mode, the rpc gets invoked after the enclave is built.
+if [ "${SGX_MODE}" == "SIM" ]; then
+    echo set check_attestation to false in SGX SIM mode
+    try ${F_SERVICEHOME}/bin/register_enclave_attestation_verification_policy.py --logfile __screen__ --loglevel WARNING
+fi
+
 sleep 5
```

Comment:
this is a little unusual. we are using a local environment variable (SGX_MODE) to capture the overall behavior of a network of eservices & the ledger. Fine for the moment, but in the future we should probably think more clearly about how to do system-wide configuration. Like SGX_MODE should be derived from the configuration of the ledger.

Reply:
thank you, per discussion. we will retain this logic until we roll out deployment "templates".